src/config/get-config.ts

```diff
@@ -0,0 +1,63 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config();
+
+export interface Config {
+  /**
+   * The discord token.
+   * This is required otherwise the validate-config function will fail.
+   */
+  TOKEN: string;
+  /**
+   * The bots prefix which should be placed in-front of
+   * every command. Defaults to '!FCC' if nothing was given
+   *  **note** this is not used yet, all command still follow the :
+   * `!myCommand` format.
+   */
+  PREFIX: string;
+  /**
+   * If the bot should log all message to the console.
+   * By default this feature is OFF to suppress the amount of
+   * logs being printed.
+   */
+  VERBOSE: boolean;
+  /**
+   * If the bot should send the FCC code of conduct via a DM
+   * to any new user who joins the server.
+   * By default this feature is OFF
+   */
+  WELCOME_DM: boolean;
```

Comment:
Quick question. This is typed as a boolean here, but compared to a string in the return. 
So my `.env` had to have it as a string and wouldn't register if I used the boolean.

Reply:
All environment variables provided from `process.env` should be strings or undefined. 
So if your using `.env` then either of the following should end up as "true":
```
WELCOME_DM=true
```
or
```
WELCOME_DM='true'
```

Reply:
`WELCOME_DM='true'` worked for me, but `WELCOME_DM=true` did not. 
I'll try testing it again to see if it was an error on my end.

Reply:
> WELCOME_DM='true' worked for me, but WELCOME_DM=true did not.
I'll try testing it again to see if it was an error on my end.

I'll recheck with `WELCOME_DM=true`

Reply:
This seems to be working now - I must have royally messed up my initial testing 😬